### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by Saphyel**
I found a bug in the templates ( templates/module/src/Form/form-alter.php.twig  &  templates/module/src/Form/form-config.php.twig ) and well I edited the others in order to use `[]` rather than `array()`

### CODE REVIEW

File: templates/module/src/Form/form-config.php.twig [hunk 1]
```diff
@@ -73,7 +73,7 @@ class {{ class_name }} extends ConfigFormBase {% endblock %}
     $config = $this->config('{{module_name}}.{{class_name_short}}');
 {% for input in inputs %}
 {% if input.fieldset is defined and input.fieldset is not empty %}
-    $form['{{ input.fieldset }}']['{{ input.name }}'] = array(
+    $form['{{ input.fieldset }}']['{{ input.name }}'] = [
```
Comment (Saphyel) [inline templates/module/src/Form/form-config.php.twig]: This is one of them.

**Comment by enzolutions** [comment]
@miguel303 please test all command with temples modified using these instructions in your local

```
git checkout -b Saphyel-templates master
git pull https://github.com/Saphyel/DrupalConsole.git templates
```

**Comment by Saphyel** [comment]
@miguel303 I didn't update the "Entities" if you want to update them here or in other PR...

**Comment by jmolivas** [comment]
@Saphyel PR was merged. This will be included on next release, Thanks.

**Pull request merged**